### ISSUE TITLE
SecurityError: localStorage is not available for opaque origins

### ISSUE THREAD

**Issue by jappareti** [mike-works/typescript-fundamentals#61]
## System Information
- Mac OS 10.13.5
- Node v9.4.0

## Describe the bug

All tests fail (not in the expected way) with the following error:

`SecurityError: localStorage is not available for opaque origins`

## To Reproduce
```
git clone https://github.com/mike-works/typescript-fundamentals tscript
cd tscript
yarn
npm test accounts 
```

 ## Expected behavior

The console should show the failing tests from the test suite, but instead I get the single error message above.

## Possible fixes

It looks like this issue is related: https://github.com/facebook/jest/issues/6766

Adding `testURL: "http://localhost/"` OR `testEnvironment: 'node'` to the `jest.config.js` file resolved the issue for me.

Another solution is to upgrade `jest` from `22.0.0` to `23.5.0` because the newest version sets the default `testURL`

**Comment by dallin-r-parker** [comment]
@mike-north sorry I didn't see this issue before making my PR.

**Comment by mike-north** [comment]
Resolved by #62
